src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -241,4 +315,14 @@ public ProcessExitCodeException(int exitCode, string fileName, string arguments)
             ExitCode = exitCode;
         }
     }
+
+    public class ProcessDataReceivedEventArgs : EventArgs
```

Comment:
Ah, I see why this is needed now. The setter is also unreachable in DataReceivedEventArgs.

Side note: `String` -> `string`